src/bootstrap.js

```diff
@@ -4,11 +4,11 @@ import app from './app';
 import env from './constants/env';
 import logger from './utils/logger';
 
-const dbConfig = env.NODE_ENV === 'production' ? {} : { dbName: 'local-project-db' };
-
 let server;
 
```

Comment:
You can add 
const db = mongoose.connection;
db.on('error', (error) => console.error(error));

after function below(in line 18)